regolith/dates.py

```diff
@@ -258,10 +258,37 @@ def is_current(thing, now=None):
         now = datetime.date.today()
     dates = get_dates(thing)
     current = False
-    if not dates['begin_date']:
-        return current
-    if not dates['end_date']:
-        dates['end_date'] = datetime.date(5000, 12, 31)
-    if dates.get("begin_date") <= now <= dates.get("end_date"):
-        current = True
-    return current
\ No newline at end of file
+    try:
+        if dates.get("begin_date") <= now <= dates.get("end_date", datetime.date(5000, 12, 31)):
+            current = True
+    except:
+        raise RuntimeError("Element that was passed did not have valid dates")
```

Comment:
good, well done!   This would be even more helpful if you can think of a way to tell the user which grant it failed on, something like
```
raise RuntimeError(f"Cannot find begin_date in document\n {thing}")
```